docs/api/index.md

```diff
@@ -1,51 +1,41 @@
 # The CiviCRM API
 
-CiviCRM has a stable comprehensive **API** (Application Programming
-Interface) that can be used to access and manage data in CiviCRM. The
-API is the recommended way for any CiviCRM extension, CMS module, or
-external program to interact with CiviCRM.
-
-Utilizing the API is superior to accessing core functions directly (e.g.
-calling raw SQL, or calling functions within the BAO files)
-because the API offers a consistent interface to CiviCRM's features. It is
-designed to function predictably with every new release so as to preserve
-backwards compatibility of the API for several versions of CiviCRM. If
-you decide to use other ways to collect data (like your own SQL statements),
-you risk running into future problems when changes to the schema and
-BAO arguments inevitably occur.
-
-The best place to begin working with the API is your own ***test*** install of
-CiviCRM, using the API explorer and the API parameter list.
+CiviCRM has a stable comprehensive **API** (Application Programming Interface) that can be used to access and manage data in CiviCRM. The API is the recommended way for any CiviCRM extension, CMS module, or external program to interact with CiviCRM.
+
+Utilizing the API is superior to accessing core functions directly (e.g.calling raw SQL, or calling functions within the BAO files) because the API offers a consistent interface to CiviCRM's features. It is designed to function predictably with every new release so as to preserve backwards compatibility of the API for several versions of CiviCRM. If you decide to use other ways to collect data (like your own SQL statements), you risk running into future problems when changes to the schema andBAO arguments inevitably occur.
```

Comment:
```suggestion
Utilizing the API is superior to accessing core functions directly (e.g.calling raw SQL, or calling functions within the BAO files) because the API offers a consistent interface to CiviCRM's features. It is designed to function predictably with every new release so as to preserve backwards compatibility of the API for several versions of CiviCRM. If you decide to use other ways to collect data (like your own SQL statements), you risk running into future problems when changes to the schema and BAO arguments inevitably occur.
```